drivers/cc110x/cc110x-rxtx.c

```diff
@@ -133,14 +154,17 @@ static void _rx_read_data(cc110x_t *dev, void(*callback)(void*), void*arg)
         else {
             DEBUG("%s:%s:%u crc-error\n", RIOT_FILE_RELATIVE, __func__, __LINE__);
             dev->cc110x_statistic.packets_in_crc_fail++;
+#if defined(MODULE_OD) && ENABLE_DEBUG
+            od_hex_dump(pkt_buf->packet.data, pkt_buf->packet.length-3,
```

Comment:
minor nit, please use spaces in math expressions, i.e., `(pkt_buf->packet.length - 3)`